# bio_paper 2022
* Rationalized deep learning super-resolution microscopy for sustained live imaging of rapid subcellular processes | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01471-3)
* Single-sequence protein structure prediction using a language model and deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01432-w)
* Real-time denoising enables high-sensitivity fluorescence time-lapse imaging beyond the shot-noise limit | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01450-8)
* Automated reconstruction of whole-embryo cell lineages by learning from sparse annotations | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01427-7)
* DeepConsensus improves the accuracy of sequences with a gap-aware sequence transformer | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01435-7)
* Deep Visual Proteomics defines single-cell identity and heterogeneity | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01302-5)
* Prediction of proteinâ€“ligand binding affinity from sequencing data with interpretable machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01307-0)
* Learning protein fitness models from evolutionary and assay-labeled data | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01146-5)
* scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01161-6)
* Using deep learning to annotate the protein universe | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01179-w)
* Multiscale PHATE identifies multimodal signatures of COVID-19 | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01186-x)
* Designing sensitive viral diagnostics with machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01213-5)
* Identification of antimicrobial peptides from the human gut microbiome using deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01226-0)
* Multi-omics single-cell data integration and regulatory inference with graph-linked embedding | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01284-4)
* Spatially informed cell-type deconvolution for spatial transcriptomics | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01273-7)
* Sparse deconvolution improves the resolution of live-cell super-resolution fluorescence microscopy | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01092-2)
* Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01094-0)
* Super-resolved spatial transcriptomics by deep data fusion | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01075-3)
* Mapping single-cell data to reference atlases by transfer learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01001-7)
* A large-scale neural network training framework for generalized estimation of single-trial population dynamics | [Nature Methods](https://www.nature.com/articles/s41592-022-01675-0)
* Detection of m6A from direct RNA sequencing using a multiple instance learning framework | [Nature Methods](https://www.nature.com/articles/s41592-022-01666-1)
* AlphaFill: enriching AlphaFold models with ligands and cofactors | [Nature Methods](https://www.nature.com/articles/s41592-022-01685-y)
* Unsupervised discovery of tissue architecture in multiplexed imaging | [Nature Methods](https://www.nature.com/articles/s41592-022-01657-2)
* Incorporating the image formation process into deep learning improves network performance | [Nature Methods](https://www.nature.com/articles/s41592-022-01652-7)
* Cellpose 2.0: how to train your own model | [Nature Methods](https://www.nature.com/articles/s41592-022-01663-4)
* SVision: a deep learning approach to resolve complex structural variants | [Nature Methods](https://www.nature.com/articles/s41592-022-01609-w)
* BIONIC: biological network integration using convolutions | [Nature Methods](https://www.nature.com/articles/s41592-022-01616-x)
* Omnipose: a high-precision morphology-independent solution for bacterial cell segmentation | [Nature Methods](https://www.nature.com/articles/s41592-022-01639-4)
* Improved AlphaFold modeling with implicit experimental information | [Nature Methods](https://www.nature.com/articles/s41592-022-01645-6)
* Annotation of spatially resolved single-cell data with STELLAR | [Nature Methods](https://www.nature.com/articles/s41592-022-01651-8)
* Self-supervised deep learning encodes high-resolution features of protein subcellular localization | [Nature Methods](https://www.nature.com/articles/s41592-022-01541-z)
* Sprod for de-noising spatially resolved transcriptomics data based on position and image information | [Nature Methods](https://www.nature.com/articles/s41592-022-01560-w)
* scBasset: sequence-based modeling of single-cell ATAC-seq using convolutional neural networks  | [Nature Methods](https://www.nature.com/articles/s41592-022-01562-8)
