# bio_paper 2022
## Journal
* Rationalized deep learning super-resolution microscopy for sustained live imaging of rapid subcellular processes | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01471-3)
* Single-sequence protein structure prediction using a language model and deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01432-w)
* Real-time denoising enables high-sensitivity fluorescence time-lapse imaging beyond the shot-noise limit | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01450-8)
* Automated reconstruction of whole-embryo cell lineages by learning from sparse annotations | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01427-7)
* DeepConsensus improves the accuracy of sequences with a gap-aware sequence transformer | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01435-7)
* Deep Visual Proteomics defines single-cell identity and heterogeneity | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01302-5)
* Prediction of protein–ligand binding affinity from sequencing data with interpretable machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01307-0)
* Learning protein fitness models from evolutionary and assay-labeled data | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01146-5)
* scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01161-6)
* Using deep learning to annotate the protein universe | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01179-w)
* Multiscale PHATE identifies multimodal signatures of COVID-19 | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01186-x)
* Designing sensitive viral diagnostics with machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01213-5)
* Identification of antimicrobial peptides from the human gut microbiome using deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01226-0)
* Multi-omics single-cell data integration and regulatory inference with graph-linked embedding | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01284-4)
* Spatially informed cell-type deconvolution for spatial transcriptomics | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01273-7)
* Sparse deconvolution improves the resolution of live-cell super-resolution fluorescence microscopy | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01092-2)
* Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01094-0)
* Super-resolved spatial transcriptomics by deep data fusion | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01075-3)
* Mapping single-cell data to reference atlases by transfer learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01001-7)
* A large-scale neural network training framework for generalized estimation of single-trial population dynamics | [Nature Methods](https://www.nature.com/articles/s41592-022-01675-0)
* Detection of m6A from direct RNA sequencing using a multiple instance learning framework | [Nature Methods](https://www.nature.com/articles/s41592-022-01666-1)
* AlphaFill: enriching AlphaFold models with ligands and cofactors | [Nature Methods](https://www.nature.com/articles/s41592-022-01685-y)
* Unsupervised discovery of tissue architecture in multiplexed imaging | [Nature Methods](https://www.nature.com/articles/s41592-022-01657-2)
* Incorporating the image formation process into deep learning improves network performance | [Nature Methods](https://www.nature.com/articles/s41592-022-01652-7)
* Cellpose 2.0: how to train your own model | [Nature Methods](https://www.nature.com/articles/s41592-022-01663-4)
* SVision: a deep learning approach to resolve complex structural variants | [Nature Methods](https://www.nature.com/articles/s41592-022-01609-w)
* BIONIC: biological network integration using convolutions | [Nature Methods](https://www.nature.com/articles/s41592-022-01616-x)
* Omnipose: a high-precision morphology-independent solution for bacterial cell segmentation | [Nature Methods](https://www.nature.com/articles/s41592-022-01639-4)
* Improved AlphaFold modeling with implicit experimental information | [Nature Methods](https://www.nature.com/articles/s41592-022-01645-6)
* Annotation of spatially resolved single-cell data with STELLAR | [Nature Methods](https://www.nature.com/articles/s41592-022-01651-8)
* Self-supervised deep learning encodes high-resolution features of protein subcellular localization | [Nature Methods](https://www.nature.com/articles/s41592-022-01541-z)
* Sprod for de-noising spatially resolved transcriptomics data based on position and image information | [Nature Methods](https://www.nature.com/articles/s41592-022-01560-w)
* scBasset: sequence-based modeling of single-cell ATAC-seq using convolutional neural networks  | [Nature Methods](https://www.nature.com/articles/s41592-022-01562-8)
* MSNovelist: de novo structure generation from mass spectra | [Nature Methods](https://www.nature.com/articles/s41592-022-01486-3)
* ScanNet: an interpretable geometric deep learning model for structure-based protein binding site prediction | [Nature Methods](https://www.nature.com/articles/s41592-022-01490-7)
* TrackMate 7: integrating state-of-the-art segmentation algorithms into tracking pipelines | [Nature Methods](https://www.nature.com/articles/s41592-022-01507-1)
* Automated high-speed 3D imaging of organoid cultures with multi-scale phenotypic quantification | [Nature Methods](https://www.nature.com/articles/s41592-022-01508-0)
* A learned embedding for efficient joint analysis of millions of mass spectra | [Nature Methods](https://www.nature.com/articles/s41592-022-01496-1)
* Multi-animal pose estimation, identification and tracking with DeepLabCut | [Nature Methods](https://www.nature.com/articles/s41592-022-01443-0)
* SLEAP: A deep learning system for multi-animal pose tracking | [Nature Methods](https://www.nature.com/articles/s41592-022-01426-1)
* VascuViz: a multimodality and multiscale imaging and visualization pipeline for vascular systems biology | [Nature Methods](https://www.nature.com/articles/s41592-021-01363-5)
* CR-I-TASSER: assemble protein structures from cryo-EM density maps using deep convolutional neural networks | [Nature Methods](https://www.nature.com/articles/s41592-021-01389-9)
* Identifying temporal and spatial patterns of variation from multimodal data using MEFISTO | [Nature Methods](https://www.nature.com/articles/s41592-021-01343-9)
* Pathway trajectory analysis with tensor imputation reveals drug-induced single-cell transcriptomic landscape | [Nature Computational Science](https://www.nature.com/articles/s43588-022-00352-8)
* Improving lab-of-origin prediction of genetically engineered plasmids via deep metric learning | [Nature Computational Science](https://www.nature.com/articles/s43588-022-00234-z)
* Large-scale microbiome data integration enables robust biomarker identification | [Nature Computational Science](https://www.nature.com/articles/s43588-022-00247-8)
* Adversarial domain translation networks for integrating large-scale atlas-level single-cell datasets | [Nature Computational Science](https://www.nature.com/articles/s43588-022-00251-y)
* Cell clustering for spatial transcriptomics data with graph neural networks | [Nature Computational Science](https://www.nature.com/articles/s43588-022-00266-5)
* Rotamer-free protein sequence design based on deep learning and self-consistency | [Nature Computational Science](https://www.nature.com/articles/s43588-022-00273-6)
* A deep manifold-regularized learning model for improving phenotype prediction from multi-modal data | [Nature Computational Science](https://www.nature.com/articles/s43588-021-00185-x)
* A multi-use deep learning method for CITE-seq and single-cell RNA-seq data integration with cell surface protein prediction and imputation | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00545-w)
* Predicting unseen antibodies’ neutralizability via adaptive graph neural networks | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00553-w)
* Predicting functional effect of missense variants using graph attention neural networks | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00561-w)
* Image prediction of disease progression for osteoarthritis by style-based manifold extrapolation | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00560-x)
* Accurate prediction of molecular properties and drug targets using a self-supervised image representation learning framework | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00557-6)
* A multilevel generative framework with hierarchical self-contrasting for bias control and transparency in structure-based ligand design | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00564-7)
* A generalizable deep learning framework for inferring fine-scale germline mutation rate maps | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00574-5)
* Detecting multiple retinal diseases in ultra-widefield fundus imaging and data-driven identification of informative regions with deep learning | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00566-5)
* Contrastive learning enables rapid mapping to multimodal single-cell atlas of multimillion scale | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00518-z)
* Accelerated rational PROTAC design via deep learning and molecular simulations | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00527-y)
* Deep neural networks with controlled variable selection for the identification of putative causal genetic variants | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00525-0)
* Transformer-based protein generation with regularized latent space optimization | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00532-1)
* scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00534-z)
* A context-aware deconfounding autoencoder for robust prediction of personalized clinical drug response from cell-line compound screening | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00541-0)
* Inferring transcription factor regulatory networks from single-cell ATAC-seq data based on graph neural networks | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00469-5)
* An adaptive graph learning method for automated molecular interactions and properties predictions | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00501-8)
* Learning biophysical determinants of cell fate with deep neural networks | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00503-6)
* Simultaneous dimensionality reduction and integration for single-cell ATAC-seq data using deep learning | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00443-1)
* Molecular contrastive learning of representations via graph neural networks | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00447-x)
* Learning functional properties of proteins with language models | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00457-9)
* A transformer-based model to predict peptide–HLA class I binding and optimize mutated peptides for vaccine design | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00459-7)
* Ad hoc learning of peptide fragmentation from mass spectra enables an interpretable detection of phosphorylated and cross-linked peptides  | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00467-7)
* Deep learning-inferred multiplex immunofluorescence for immunohistochemical image quantification | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00471-x)
* Multiscale simulations of complex systems by learning their effective dynamics | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-022-00464-w)
* Hierarchical deep reinforcement learning reveals a modular mechanism of cell movement | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-021-00431-x)
* Interpreting neural networks for biological sequences by learning stochastic masks | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-021-00428-6)
* Geometry-enhanced molecular representation learning for property prediction | [Nature Machine Intelligence](https://www.nature.com/articles/s42256-021-00438-4)
* Spatial-ID: a cell typing method for spatially resolved transcriptomics via transfer learning and spatial embedding | [Nature Communications][https://www.nature.com/articles/s41467-022-35288-0](https://www.nature.com/articles/s41467-022-35288-0)
* Graph-based autoencoder integrates spatial transcriptomics with chromatin images and identifies joint biomarkers for Alzheimer’s disease | [Nature Communications](https://www.nature.com/articles/s41467-022-35233-1)
* pGlycoQuant with a deep residual network for quantitative glycoproteomics at intact glycopeptide level | [Nature Communications](https://www.nature.com/articles/s41467-022-35172-x)
* ProtGPT2 is a deep unsupervised language model for protein design | [Nature Communications](https://www.nature.com/articles/s41467-022-32007-7)
* iDNA-ABF: multi-scale deep biological language learning model for the interpretable prediction of DNA methylations | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02780-1)
* KAGE: fast alignment-free graph-based genotyping of SNPs and short indels | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02771-2)
* BASS: multi-scale and multi-sample analysis enables accurate cell type clustering and spatial domain detection in spatial transcriptomic studies | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02734-7)
* A benchmark study of deep learning-based multi-omics data fusion methods for cancer | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02739-2)
* BindVAE: Dirichlet variational autoencoders for de novo motif discovery from accessible chromatin | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02723-w)
* Integrating temporal single-cell gene expression modalities for trajectory inference and disease prediction | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02749-0)
* Bi-order multimodal integration of single-cell data  | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02679-x)
* Identifying common transcriptome signatures of cancer by interpreting deep learning models | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02681-3)
* SeqScreen: accurate and sensitive functional screening of pathogenic sequences via ensemble learning | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02695-x)
* scDART: integrating unmatched scRNA-seq and scATAC-seq data and learning cross-modality relationship simultaneously | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02706-x)
* scSTEM: clustering pseudotime ordered single-cell data | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02716-9)
* A deep generative model for multi-view profiling of single-cell RNA-seq and ATAC-seq data | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02595-6)
* Universal prediction of cell-cycle position using transfer learning | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02581-y)
* Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data | [Genome Biology](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02622-0)



* Ontology-aware deep learning enables ultrafast and interpretable source tracking among sub-million microbial community samples from hundreds of niches | [Genome Medicine](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-022-01047-5)
* 
* Learning the protein language: Evolution, structure, and function | [Cell systems](https://www.sciencedirect.com/science/article/pii/S2405471221002039)
## Meeting
