# bio_paper
papers of the bioinformatics.

## 2022
* Rationalized deep learning super-resolution microscopy for sustained live imaging of rapid subcellular processes | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01471-3)
* Single-sequence protein structure prediction using a language model and deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01432-w)
* Real-time denoising enables high-sensitivity fluorescence time-lapse imaging beyond the shot-noise limit | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01450-8)
* Automated reconstruction of whole-embryo cell lineages by learning from sparse annotations | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01427-7)
* DeepConsensus improves the accuracy of sequences with a gap-aware sequence transformer | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01435-7)
* Deep Visual Proteomics defines single-cell identity and heterogeneity | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01302-5)
* Prediction of proteinâ€“ligand binding affinity from sequencing data with interpretable machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01307-0)
* Learning protein fitness models from evolutionary and assay-labeled data | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01146-5)
* scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01161-6)
* Using deep learning to annotate the protein universe | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01179-w)
* Multiscale PHATE identifies multimodal signatures of COVID-19 | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01186-x)
* Designing sensitive viral diagnostics with machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01213-5)
* Identification of antimicrobial peptides from the human gut microbiome using deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01226-0)
* Multi-omics single-cell data integration and regulatory inference with graph-linked embedding | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01284-4)
* Spatially informed cell-type deconvolution for spatial transcriptomics | [Nature Biotechnology](https://www.nature.com/articles/s41587-022-01273-7)
* Sparse deconvolution improves the resolution of live-cell super-resolution fluorescence microscopy | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01092-2)
* Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01094-0)
* Super-resolved spatial transcriptomics by deep data fusion | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01075-3)
* *Mapping single-cell data to reference atlases by transfer learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-01001-7)

## 2021
* Prediction of drug efficacy from transcriptional profiles with deep learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-021-00946-z)
* Improved metagenome binning and assembly using deep variational autoencoders | [Nature Biotechnology](https://www.nature.com/articles/s41587-020-00777-4)
* Deep diversification of an AAV capsid protein by machine learning | [Nature Biotechnology](https://www.nature.com/articles/s41587-020-00793-4)
## 2019

## 2018
